domains/Discussions/Tests/Feature/QuestionsUpdateTest.php

```diff
@@ -14,7 +14,7 @@
 use Laravel\Lumen\Testing\DatabaseMigrations;
 use Tests\TestCase;
 
-class QuestionsUpdateTest extends TestCase
+class QuestionsDeleteTest extends TestCase
```

Comment:
Why is this class name different than the file name? Is this intended?

Reply:
Obviously not.